docs/source/kedro_project_setup/configuration.md

```diff
@@ -29,6 +29,27 @@ This recursively scans for configuration files firstly in the `conf/base/` (`bas
   * file is located in a sub-directory whose name is prefixed with `catalog`
 * *And* file extension is one of the following: `yaml`, `yml`, `json`, `ini`, `pickle`, `xml` or `properties`
 
+This logic is specified by `config_patterns` in the [ConfigLoader](/kedro.config.ConfigLoader) and [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) classes. By default those patterns are set as follows for the configuration of catalog, parameters, logging and credentials:
```

Comment:
```suggestion
This logic is specified by `config_patterns` in the [ConfigLoader](/kedro.config.ConfigLoader) and [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) classes. By default those patterns are set as follows for the configuration of catalog, parameters, logging and credentials:

```

Reply:
Adding an extra line. In past releases, not leaving a line between the text and code caused rendering issues. I'm not sure if that's still the case but it's probably a good idea to be safe.